eap74-openjdk8-rhel8-image-stream.json

```diff
@@ -0,0 +1,135 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap74-openjdk8-rhel8-openshift-image-streams",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform 7.4.0 with OpenJDK 8 and RHEL 8",
```

Comment:
I don't really like us having the base OS image name in here, as we could just move it to UBI 9 and everything would still likely be the same for customers, so I'd suggest only having it in the imagestream filename and this first description. The description, I'd suggest read: ""ImageStream definition for JBoss Enterprise Application Platform 7.4.0 with OpenJDK 8 based on UBI 8"